emulate-browsers/base/injected-scripts/MediaRecorder.isTypeSupported.ts

```diff
@@ -1,7 +1,7 @@
 const { supportedCodecs } = args;
 
 if ((window as any).MediaRecorder) {
-  proxyFunction((window as any).MediaRecorder, 'isTypeSupported', (func, thisArg, type) => {
+  proxyFunction((window as any).MediaRecorder, 'isTypeSupported', (func, thisArg, [type]) => {
```

Comment:
Does this blow up if you call MediaRecorder.isTypeSupported() with no params?

Reply:
No